backend.py

```diff
@@ -20,13 +20,15 @@ def __init__(self, direction, coordinates, name):
         self.direction = direction
         self.coordinates = coordinates
         self.start_coordinates = coordinates
-        self.program = []
+        self.program = [None, None, None, None, None]
         self.lives = 3
         self.flags = 0
         self.damages = 0
         self.power_down = False
         self.name = name
         self.selection_confirmed = False
+        self.unblocked_cards = 5
+        self.blocked_cards = []
```

Comment:
Blokované karty doporučuju mít jen na jednom místě – buď v `blocked_cards` nebo v `program`, ne v obou dvou najednou.

Reply:
hmm tak to nevím jak udělám, protože ty blocked_cards posílám zvlášť klientovi. Jedině tedy na začátku vyzobat z toho programu nějakou metodou a poslat a nebudu je teda dávat tady jako atribut.